packages/office-ui-fabric-react/src/components/ContextualMenu/ContextualMenuItemChildren.types.ts

```diff
@@ -0,0 +1,12 @@
+import { ContextualMenuItemChildren } from './ContextualMenuItemChildren';
+import { IContextualMenuItem } from './ContextualMenu.types';
+import { IMenuItemClassNames } from './ContextualMenu.classNames';
+
+export interface IContextualMenuItemChildrenProps
+  extends React.Props<IContextualMenuItemChildrenProps> {
```

Comment:
Use React.HTMLAttributes<div> instead. Really we should be using that everywhere. That also, potentially, lets consumers mix in normal div props.